extensions/sql-bindings/src/common/constants.ts

```diff
@@ -75,6 +75,10 @@ export const userPasswordLater = localize('userPasswordLater', 'In order to user
 export const openFile = localize('openFile', "Open File");
 export const closeButton = localize('closeButton', "Close");
 export const connectionProgressTitle = localize('connectionProgressTitle', "Testing SQL Server connection...");
+export const selectObjectName = localize('selectObjectName', 'Select object name method');
+export const enterObjectName = localize('enterObjectName', 'Enter SQL table or view to query');
```

Comment:
Should we be asking about views if views aren't fully supported yet?

Reply:
(same with the similar string for input bindings)

Reply:
Yeah, users can still utilize views by manually entering the object name and will only be shown this message for input bindings:
`Enter SQL table or view to query`.